### ISSUE TITLE
Network.enable() was not found. With Node 7.8

### ISSUE THREAD

**Issue by nazuna1** [cyrus-and/chrome-remote-interface#215]
| Component | Version
|-|-
| Operating system	  	 	| CentOS 7
| Node.js					| 7.8
| Chrome/Chromium/...		| WRITE HERE
| chrome-remote-interface	| WRITE HERE

I was copy and run https://github.com/cyrus-and/chrome-remote-interface#sample-api-usage
but i saw message below
![nodejs-chrome-remote-inspecter-error](https://user-images.githubusercontent.com/18659202/28572804-7855f8ce-7183-11e7-962c-c03ea4fe04d9.PNG)

i don't know this happen.
found this wiki, and saw #156 
#156 is node 6.10, my node ver 7.8, pls help...

**Comment by cyrus-and** [comment]
Yes that's the same scenario as #156, even if your Node.js version is 7.8 and this [comment][1] should explain why.

What are you really trying to achieve?

[1]: https://github.com/cyrus-and/chrome-remote-interface/issues/156#issuecomment-307063861

**Comment by nazuna1** [comment]
> What are you really trying to achieve?

I just want to run the samples you have provided and learn how to use them.

**Comment by cyrus-and** [comment]
So you need to connect to Chrome/Chromium and not to Node.js, see instructions [here][1]. You can actually inspect/debug a Node.js process but it's a niche feature, an example is [here][2] if you're interested.

[1]: https://github.com/cyrus-and/chrome-remote-interface#chromechromium
[2]: https://github.com/cyrus-and/chrome-remote-interface/wiki/Connect-to-Node.js

**Comment by nazuna1** [comment]
thnx :) but, run google-chrome( [here](https://github.com/cyrus-and/chrome-remote-interface/wiki/Connect-to-Node.js) ), i saw message below
![google-chrome_headless_error](https://user-images.githubusercontent.com/18659202/28574526-35194f06-7189-11e7-8ba2-523e7e9c76dc.PNG)

I tried connecting to Chrome as below.

> google-chrome --remote-debugging-port=9222

but not to run them.

> Network.enable()

> Page.enable()

**Comment by cyrus-and** [comment]
That error seems about a bogus Chrome installation, try running it like this:

    google-chrome --remote-debugging-port=9222 --headless --disable-gpu

Once you manage to make it work, you can just try to run the examples.

**Comment by nazuna1** [comment]
`google-chrome --remote-debugging-port=9222 --headless --disable-gpu`

Same Error...

**Comment by cyrus-and** [comment]
I just noticed that you edited your [comment][1], look there are two problems here:

1. you cannot use `Page.enable` or any other domain which is not in the [V8][2] protocol when you are inspecting Node.js (and anyway it's not what you want);

2. your Chrome setup seems to have some shared libraries issue, make sure you can properly run Chrome before attempting to inspect it.

Here's a sample session:

- in one terminal start:

        google-chrome --remote-debugging-port=9222 --headless --disable-gpu

- in another run:

        node example.js

where `example.js` is the [main example][3].

[1]: https://github.com/cyrus-and/chrome-remote-interface/issues/215#issuecomment-317744472
[2]: https://chromedevtools.github.io/debugger-protocol-viewer/v8/
[3]: https://github.com/cyrus-and/chrome-remote-interface#sample-api-usage

**Comment by cyrus-and** [comment]
Did you get any further with this?
